src/cdk/portal/portal-directives.ts

```diff
@@ -181,7 +181,9 @@ export class CdkPortalOutlet extends BasePortalOutlet implements OnInit, OnDestr
    */
   attachTemplatePortal<C>(portal: TemplatePortal<C>): EmbeddedViewRef<C> {
     portal.setAttachedHost(this);
-    const viewRef = this._viewContainerRef.createEmbeddedView(portal.templateRef, portal.context);
+    const viewRef = this._viewContainerRef.createEmbeddedView(portal.templateRef, portal.context, {
+      injector: portal.injector,
+    });
```

Comment:
Should it be
```ts
    const viewRef = this._viewContainerRef.createEmbeddedView(
      portal.templateRef,
      portal.context, portal.injector ? { injector: portal.injector } : undefined
    );
```
instead?

See [StackBlitz](https://stackblitz.com/edit/angular-mnrvoq?file=package.json). When we click **Render template portal** button it throws an error:
![image](https://user-images.githubusercontent.com/2744696/161652599-7b94ffe1-b652-4a99-90a8-55fd847fd67e.png)

Reply:
I can't reproduce this error locally in our dev app. I think the problem is that your Stackblitz is running v14 of the CDK against v13 of the Framework which doesn't support the `injector` option yet.

Reply:
@crisbeto, right. Sorry about that.